maintainers/maintainer-list.nix

```diff
@@ -13771,6 +13771,15 @@
     githubId = 245573;
     name = "Dmitry Kalinkin";
   };
+  victormignot = {
```

Comment:
Please add yourself to maintainer-list as separate commit.

Reply:
Done. Sorry for the mistake !